.travis.yml

```diff
@@ -55,6 +55,20 @@ jobs:
       os: linux
       jdk: openjdk13
       env: CHECK_RUST=false
+    - name: "Linux JDK EA CHECK_RUST=false"
+      os: linux
+      jdk: openjdk-ea
+      env: CHECK_RUST=false
+      before_script:
+        - if [[ "$TRAVIS_EVENT_TYPE" != "cron" ]]; then travis_terminate 0; fi
```

Comment:
Can this job filtering be accomplished with the [conditional jobs](https://docs.travis-ci.com/user/conditional-builds-stages-jobs) (namely, `type = cron` [condition](https://docs.travis-ci.com/user/conditions-v1))? Would be more idiomatic, IMO.

Reply:
That would be brilliant, because just firing the VM and performing before_install takes about 2 min (x 3).